docs/File-Formats.md

```diff
@@ -1599,6 +1600,31 @@ Allowed values for Arm-level copy-number data are `Loss`, `Gain`, and `Unchanged
 
 Please find example file format here: [Meta file example](https://github.com/cBioPortal/cbioportal-frontend/blob/master/end-to-end-test/local/studies/lgg_ucsf_2014_test_generic_assay/meta_armlevel_CNA.txt) and [Data file example](https://github.com/cBioPortal/cbioportal-frontend/blob/master/end-to-end-test/local/studies/lgg_ucsf_2014_test_generic_assay/data_armlevel_CNA.txt)
 
+### Mutational Signature Data
+Mutational Signature data is a predefined subtype of Generic Assay Data. Setting `generic_assay_type: MUTATIONAL_SIGNATURE` 
+in the meta file will make cBioPortal interpret the data as Mutational Signature data.
+
+#### Mutational Signature meta files
+The mutational signature meta files follow the same convention as the [Generic Assay Meta file](#generic-assay-meta-file),
+however there are some key differences:
+- `genetic_assay_type` should be set to `MUTATIONAL_SIGNATURE`
+- `stable_id` values should end with: `{datatype}_{identifier}`, where:
+  - `datatype` is one of `contribution`, `pvalue` or `matrix`
```

Comment:
`datatype` can either be `contribution`, `pvalue` or `matrix`. (Might be writing preference)